docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -6,6 +6,16 @@ The Oracle connector allows querying and creating tables in an external Oracle
 database. Connectors let Trino join data provided by different databases,
 like Oracle and Hive, or different Oracle database instances.
 
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Oracle data
+source are:
+
+* Oracle ?? or higher
```

Comment:
12?

Reply:
actually perhaps latest 12?

Reply:
I went with just

Oracle 12 or higher

Not sure how this works with the numerous digits and such on the Oracle versioning, latest is probably too narrow imho .. also we probably dont really test latest all the time